src/table/components/__tests__/withSelections.spec.jsx

```diff
@@ -68,16 +67,9 @@ describe('withSelections', () => {
     );
     fireEvent.mouseUp(queryByText(value));
 
-    expect(selectionsUtils.selectCell).toHaveBeenCalledTimes(1);
-    expect(selectionsUtils.selectCell).toHaveBeenCalledWith(
```

Comment:
I don't see the point of checking what has been called with if it is anything. The main point here is to test whether it has been called or not